dev/test-studio/sanity.cli.ts

```diff
@@ -13,12 +27,34 @@ export default defineCliConfig({
   // A) `SANITY_STUDIO_REACT_STRICT_MODE=false pnpm dev`
   // B) creating a `.env` file locally that sets the same env variable as above
   reactStrictMode: true,
-  reactCompiler: {target: '18'},
+  reactCompiler: millionLintEnabled
+    ? {
+        target: '18',
+        sources: (filename) => {
+          if (filename.includes('node_modules')) {
+            return false
+          }
+          return millionInclude.every(
+            (pattern) => !filename.endsWith(`/${pattern.split('**/')[1]}`),
+          )
+        },
+      }
+    : {target: '18'},
   vite(viteConfig: UserConfig): UserConfig {
     const reactProductionProfiling = process.env.REACT_PRODUCTION_PROFILING === 'true'
 
     return {
       ...viteConfig,
+      plugins: millionLintEnabled
+        ? [
+            require('@million/lint').vite({
```

Comment:
I tried to use it as a regular import at the top originally.
But for some reason, if you have this line top-level:
```ts
import Million from '@million/lint'
```
then `sanity build` [hangs forever](https://vercel.com/sanity-sandbox/test-studio/8ZbBMVR6PbKGF2tTrMKmymytrjf3?filter=errors)

Reply:
Yep, I'll add a comment. It's the WebSocket server setup for the VS Code extension that isn't closed 🙌